crates/ironrdp-cliprdr-native/src/windows.rs

```diff
@@ -112,18 +116,53 @@ pub struct WinClipboard {
 
 impl WinClipboard {
     /// Creates new clipboard instance.
-    ///
-    /// # Safety
-    ///
-    /// `window` must be a valid window handle to safely initialize clipboard processing.
-    /// The handle should be alive during the whole lifetime of the constructed clipboard instance.
-    pub unsafe fn new(window: HWND, message_proxy: impl ClipboardMessageProxy + 'static) -> WinCliprdrResult<Self> {
+    pub fn new(message_proxy: impl ClipboardMessageProxy + 'static) -> WinCliprdrResult<Self> {
```

Comment:
suggestion: Maybe add a little bit more of documentation about the hidden window pattern?

```suggestion
    ///
    /// Under the hood, a hidden window is created for capturing the clipboard events.
    pub fn new(message_proxy: impl ClipboardMessageProxy + 'static) -> WinCliprdrResult<Self> {
```

Reply:
ok

Reply:
done